var/ramble/repos/builtin/package_managers/pip/package_manager.py

```diff
@@ -359,6 +360,14 @@ def copy_from_external_env(self, external_env_path):
         and writes the output requirements.txt to the current env_path.
         """
         self._check_env_configured()
+        dest = os.path.join(self.env_path, self._requirement_file_name)
+        # When a file is given, assume it's a requirements.txt.
+        if os.path.isfile(external_env_path):
```

Comment:
I don't see it in this PR, so I'm asking just to be sure.

Is `external_env_path` expanded using the expander? (i.e. can this be input as a variable from an experiment?)

Reply:
Right now it's not expanded. I will add it.

Reply:
Added.